### PR TITLE
Re-names Head of Security to Chief of Security

### PR DESCRIPTION
**Pull request by KingOfThePing**
No one can stop me. 

This has been discussed a long time, and many, if not most, agree that "Head of Security" is a strange title and not a "real" job. With the start of NBT this is probably the best and only moment, to rename this role to what it always should have been: 

"Chief of Security"

No longer is it named Head of Security, but called Commander, no longer does it sound weird. Chief of Security, that's it.

### CODE REVIEW

**Comment by KingOfThePing** [comment]
!review

File: code/game/objects/items/devices/radio/headset.dm [hunk 1]
```diff
@@ -537,24 +537,24 @@
 	ks2type = /obj/item/device/encryptionkey/headset_penal
 
 /obj/item/device/radio/headset/heads/hos
-	name = "head of security's headset"
+	name = "Chief of Security's headset"
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/radio/headset.dm:540]: Inconsistent capitalization with other headsets; should be all lowercase

File: code/game/objects/items/toys.dm [hunk 2]
```diff
@@ -760,8 +760,8 @@
 	icon_state = "hop"
 
 /obj/item/toy/figure/hos
-	name = "head of security action figure"
-	desc = "A \"Space Life\" brand head of security action figure."
+	name = "Chief of Security action figure"
```
Comment (NonQueueingMatt) [inline code/game/objects/items/toys.dm:763]: Ditto

File: code/game/objects/structures/bedsheet_bin.dm [hunk 3]
```diff
@@ -288,7 +288,7 @@ LINEN BINS
 	item_state = "sheetce"
 
 /obj/item/bedsheet/hos
-	name = "head of security's bedsheet"
+	name = "Chief of Security's bedsheet"
```
Comment (NonQueueingMatt) [inline code/game/objects/structures/bedsheet_bin.dm:291]: Ditto

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 4]
```diff
@@ -107,7 +107,7 @@
 	new /obj/item/clothing/shoes/white(src)
 
 /obj/structure/closet/secure_closet/hos
-	name = "head of security's locker"
+	name = "Chief of Security's locker"
```
Comment (NonQueueingMatt) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm:110]: Ditto

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 5]
```diff
@@ -156,7 +156,7 @@
 	new /obj/item/storage/box/fancy/keypouch/sec(src)
 
 /obj/structure/closet/secure_closet/hos2
-	name = "head of security's attire"
+	name = "Chief of Security's attire"
```
Comment (NonQueueingMatt) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm:159]: Ditto

File: code/modules/clothing/head/helmet.dm [hunk 6]
```diff
@@ -53,7 +53,7 @@
 		to_chat(user, "This helmet has a built-in camera. It's [!ispath(camera) && camera.status ? "" : "in"]active.")
 
 /obj/item/clothing/head/helmet/hos
-	name = "head of security helmet"
+	name = "Chief of Security helmet"
```
Comment (NonQueueingMatt) [inline code/modules/clothing/head/helmet.dm:56]: Ditto

File: code/modules/clothing/head/jobs.dm [hunk 7]
```diff
@@ -178,8 +178,8 @@
 	item_state = "hos_hat"
 
 /obj/item/clothing/head/hos/corp
-	name = "corporate head of security hat"
-	desc = "The black parade hat of the Head of Security. For showing the officers who's in charge."
+	name = "corporate Chief of Security hat"
```
Comment (NonQueueingMatt) [inline code/modules/clothing/head/jobs.dm:181]: Ditto

File: code/modules/clothing/head/jobs.dm [hunk 8]
```diff
@@ -164,8 +164,8 @@
 	contained_sprite = TRUE
 
 /obj/item/clothing/head/hos
-	name = "head of security hat"
-	desc = "The navy blue parade hat of the Head of Security. For showing the officers who's in charge."
+	name = "Chief of Security hat"
```
Comment (NonQueueingMatt) [inline code/modules/clothing/head/jobs.dm:167]: Ditto

File: code/modules/clothing/suits/jobs.dm [hunk 9]
```diff
@@ -134,7 +134,7 @@ obj/item/clothing/suit/apron/overalls/blue
 	item_state = "wardendnavyjacket"
 
 /obj/item/clothing/suit/storage/security/hos
-	name = "head of security's jacket"
+	name = "Chief of Security's jacket"
```
Comment (NonQueueingMatt) [inline code/modules/clothing/suits/jobs.dm:137]: Ditto

File: code/modules/clothing/under/accessories/accessory.dm [hunk 10]
```diff
@@ -473,8 +473,8 @@
 	item_state = "capcloak"
 
 /obj/item/clothing/accessory/poncho/roles/cloak/hos
-	name = "head of security's cloak"
-	desc = "An elaborate cloak meant to be worn by the Head of Security."
+	name = "Chief of Security's cloak"
```
Comment (NonQueueingMatt) [inline code/modules/clothing/under/accessories/accessory.dm:476]: Ditto

File: code/modules/clothing/under/jobs/security.dm [hunk 11]
```diff
@@ -150,11 +150,11 @@
...
 
 /*
- * Head of Security
+ * Chief of Security
  */
 /obj/item/clothing/under/rank/head_of_security
-	desc = "It's a jumpsuit worn by those few with the dedication to achieve the position of \"Head of Security\". It has additional armor to protect the wearer."
-	name = "head of security's uniform"
+	desc = "It's a jumpsuit worn by those few with the dedication to achieve the position of \"Chief of Security\". It has additional armor to protect the wearer."
+	name = "Chief of Security's uniform"
```
Comment (NonQueueingMatt) [inline code/modules/clothing/under/jobs/security.dm:157]: Ditto

File: code/modules/clothing/under/miscellaneous.dm [hunk 12]
```diff
@@ -342,15 +342,15 @@
 	worn_state = "crdress"
 
 /obj/item/clothing/under/hosformalmale
-	name = "head of security's formal uniform"
-	desc = "A male head of security's formal-wear, for special occasions."
+	name = "Chief of Security's formal uniform"
```
Comment (NonQueueingMatt) [inline code/modules/clothing/under/miscellaneous.dm:345]: Ditto

File: code/modules/clothing/under/miscellaneous.dm [hunk 13]
```diff
@@ -342,15 +342,15 @@
...
+	desc = "A male Chief of Security's formal-wear, for special occasions."
 	icon_state = "hos_formal_male"
 	item_state = "r_suit"
 	worn_state = "hos_formal_male"
 
 /obj/item/clothing/under/hosformalfem
-	name = "head of security's formal uniform"
-	desc = "A female head of security's formal-wear, for special occasions."
+	name = "Chief of Security's formal uniform"
```
Comment (NonQueueingMatt) [inline code/modules/clothing/under/miscellaneous.dm:352]: Ditto

File: code/modules/modular_computers/computers/subtypes/preset_console.dm [hunk 14]
```diff
@@ -101,7 +101,7 @@
 	enrolled = 1
 
 /obj/item/modular_computer/console/preset/security/hos
-	name = "head of security's console"
+	name = "Chief of Security's console"
```
Comment (NonQueueingMatt) [inline code/modules/modular_computers/computers/subtypes/preset_console.dm:104]: Ditto

File: code/modules/modular_computers/computers/subtypes/preset_laptop.dm [hunk 15]
```diff
@@ -97,8 +97,8 @@
 	enrolled = 1
 
 /obj/item/modular_computer/laptop/preset/security/hos
-	name = "head of security's laptop"
-	desc = "A portable computer belonging to the head of security. It smells faintly of gunpowder."
+	name = "Chief of Security's laptop"
```
Comment (NonQueueingMatt) [inline code/modules/modular_computers/computers/subtypes/preset_laptop.dm:100]: Ditto

File: code/modules/paperwork/stamps.dm [hunk 16]
```diff
@@ -20,7 +20,7 @@
 	icon_state = "stamp-hop"
 
 /obj/item/stamp/hos
-	name = "head of security's rubber stamp"
+	name = "Chief of Security's rubber stamp"
```
Comment (NonQueueingMatt) [inline code/modules/paperwork/stamps.dm:23]: Ditto

File: code/game/objects/items/devices/radio/headset.dm [hunk 17]
```diff
@@ -537,24 +537,24 @@
...
 
 /obj/item/device/radio/headset/heads/hos/alt
-	name = "head of security's bowman headset"
+	name = "Chief of Security's bowman headset"
 	icon_state = "hos_headset_alt"
 	item_state = "headset_alt"
 
 /obj/item/device/radio/headset/alt/double/hos
-	name = "head of security's soundproof headset"
+	name = "Chief of Security's soundproof headset"
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/radio/headset.dm:551]: Ditto

File: code/game/objects/items/devices/radio/headset.dm [hunk 18]
```diff
@@ -537,24 +537,24 @@
...
 /obj/item/device/radio/headset/heads/hos
-	name = "head of security's headset"
+	name = "Chief of Security's headset"
 	desc = "The headset of the man who protects your worthless lifes."
 	icon_state = "hos_headset"
 	ks2type = /obj/item/device/encryptionkey/heads/hos
 
 /obj/item/device/radio/headset/heads/hos/alt
-	name = "head of security's bowman headset"
+	name = "Chief of Security's bowman headset"
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/radio/headset.dm:546]: Ditto

File: code/game/objects/items/devices/radio/headset.dm [hunk 19]
```diff
@@ -537,24 +537,24 @@
...
 /obj/item/device/radio/headset/alt/double/hos
-	name = "head of security's soundproof headset"
+	name = "Chief of Security's soundproof headset"
 	icon_state = "earset_HoS"
 	item_state = "earset_HoS"
 	ks2type = /obj/item/device/encryptionkey/heads/hos
 
 /obj/item/device/radio/headset/wrist/hos
-	name = "head of security's wristbound radio"
+	name = "Chief of Security's wristbound radio"
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/radio/headset.dm:557]: Ditto

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/objective.dm [hunk 20]
```diff
@@ -431,7 +431,7 @@ datum/objective/steal
 		"a research director's jumpsuit" = /obj/item/clothing/under/rank/research_director,
 		"a chief engineer's jumpsuit" = /obj/item/clothing/under/rank/chief_engineer,
 		"a chief medical officer's jumpsuit" = /obj/item/clothing/under/rank/chief_medical_officer,
-		"a head of security's jumpsuit" = /obj/item/clothing/under/rank/head_of_security,
+		"a Chief of Security's jumpsuit" = /obj/item/clothing/under/rank/head_of_security,
```
Comment (mikomyazaki) [inline code/game/gamemodes/objective.dm:434]: Ditto.

File: code/game/jobs/job/security.dm [hunk 21]
```diff
@@ -79,7 +79,7 @@
 	faction = "Station"
 	total_positions = 1
 	spawn_positions = 1
-	supervisors = "the head of security"
+	supervisors = "the Chief of Security"
```
Comment (mikomyazaki) [inline code/game/jobs/job/security.dm:82]: Ditto.

File: code/game/jobs/job/security.dm [hunk 22]
```diff
@@ -136,7 +136,7 @@
 	faction = "Station"
 	total_positions = 2
 	spawn_positions = 2
-	supervisors = "the head of security"
+	supervisors = "the Chief of Security"
```
Comment (mikomyazaki) [inline code/game/jobs/job/security.dm:139]: Ditto.

File: code/game/jobs/job/security.dm [hunk 23]
```diff
@@ -190,7 +190,7 @@
 	faction = "Station"
 	total_positions = 4
 	spawn_positions = 4
-	supervisors = "the head of security"
+	supervisors = "the Chief of Security"
```
Comment (mikomyazaki) [inline code/game/jobs/job/security.dm:193]: Ditto.

File: code/game/jobs/job/security.dm [hunk 24]
```diff
@@ -246,7 +246,7 @@
 	faction = "Station"
 	total_positions = 2
 	spawn_positions = 2
-	supervisors = "the Head of Security"
+	supervisors = "the Chief of Security"
```
Comment (mikomyazaki) [inline code/game/jobs/job/security.dm:249]: Ditto.

File: code/game/machinery/suit_cycler.dm [hunk 25]
```diff
@@ -118,10 +118,10 @@
 	can_repair = TRUE
 
 /obj/machinery/suit_cycler/hos
-	name = "head of security suit cycler"
-	model_text = "head of Security"
+	name = "Chief of Security suit cycler"
```
Comment (mikomyazaki) [inline code/game/machinery/suit_cycler.dm:121]: Ditto.

File: code/game/machinery/suit_cycler.dm [hunk 26]
```diff
@@ -118,10 +118,10 @@
...
+	model_text = "Chief of Security"
```
Comment (mikomyazaki) [inline code/game/machinery/suit_cycler.dm:122]: Ditto.

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 27]
```diff
@@ -120,18 +120,18 @@
 	gear_tweaks += new /datum/gear_tweak/path(wardenhead)
 
 /datum/gear/head/hos
-	display_name = "headwear, security (head of security)"
-	description = "A selection of head of security headwear."
+	display_name = "headwear, security (Chief of Security)"
```
Comment (mikomyazaki) [inline code/modules/client/preference_setup/loadout/loadout_head.dm:123]: Ditto.

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 28]
```diff
@@ -120,18 +120,18 @@
...
 	..()
 	var/list/hoshead = list()
-	hoshead["head of security hat, standard"] = /obj/item/clothing/head/hos
-	hoshead["head of security hat, corporate"] = /obj/item/clothing/head/hos/corp
-	hoshead["head of security beret, standard"] = /obj/item/clothing/head/beret/security/hos
-	hoshead["head of security beret, corporate"] = /obj/item/clothing/head/beret/security/hos/corp
+	hoshead["Chief of Security hat, standard"] = /obj/item/clothing/head/hos
+	hoshead["Chief of Security hat, corporate"] = /obj/item/clothing/head/hos/corp
+	hoshead["Chief of Security beret, standard"] = /obj/item/clothing/head/beret/security/hos
+	hoshead["Chief of Security beret, corporate"] = /obj/item/clothing/head/beret/security/hos/corp
```
Comment (mikomyazaki) [inline code/modules/client/preference_setup/loadout/loadout_head.dm:134]: Ditto.

File: code/modules/clothing/under/jobs/security.dm [hunk 29]
```diff
@@ -150,11 +150,11 @@
 	contained_sprite = TRUE
 
 /*
- * Head of Security
+ * Chief of Security
  */
 /obj/item/clothing/under/rank/head_of_security
```
Comment (mikomyazaki) [inline code/modules/clothing/under/jobs/security.dm:155]: Idk if it matters. But these typepaths will now be misleading. We changed all the HoP ones to XO for the NBT I think?

File: html/changelog.html [hunk 30]
```diff
@@ -8769,11 +8769,11 @@ <h3 class="author">Snakebittenn updated:</h3>
 			<h2 class="date">11 September 2020</h2>
 			<h3 class="author">Aboshehab, Stryker, Wowzewow updated:</h3>
 			<ul class="changes bgimages16">
-				<li class="rscadd">Added a black beret for the Head of Security, courtesy of Stryker. Now found in the loadout and relevant lockers.</li>
```
Comment (mikomyazaki) [inline html/changelog.html:8772]: You don't need to change old changelogs.

File: html/changelogs/.all_changelog.yml [hunk 31]
```diff
@@ -2598,7 +2598,7 @@ DO NOT EDIT THIS FILE BY HAND!  AUTOMATICALLY GENERATED BY ss13_genchangelog.py.
   - tweak: Replaced most of the references of the Nyx system with Tau Ceti.
   - tweak: Voidsuits, armor and armored uniforms should have a better resistance to
       taser and baton hits.
-  - tweak: Head of security armor's options were tweaked to be more like each other.
```
Comment (mikomyazaki) [inline html/changelogs/.all_changelog.yml:2601]: You don't need to change old changelogs.

**Review by mikomyazaki — CHANGES_REQUESTED** [review]

File: html/changelogs/.all_changelog.yml (hunk 31 above)
Comment (KingOfThePing) [reply html/changelogs/.all_changelog.yml:2601]: I have no idea how this got changed. Lmao.

**Comment by KingOfThePing** [comment]
I have also repathed the old `/hos` path to `/cos` for sanity's sake in the long run.

File: code/game/gamemodes/objective.dm (hunk 20 above)
Comment (SleepyGemmy) [inline code/game/gamemodes/objective.dm:434]:
```suggestion
		"a chief of security's jumpsuit" = /obj/item/clothing/under/rank/head_of_security,
```

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 32]
```diff
@@ -119,19 +119,19 @@
 	wardenhead["warden beret, corporate"] = /obj/item/clothing/head/beret/security/warden/corp
 	gear_tweaks += new /datum/gear_tweak/path(wardenhead)
 
-/datum/gear/head/hos
-	display_name = "headwear, security (head of security)"
-	description = "A selection of head of security headwear."
+/datum/gear/head/cos
+	display_name = "headwear, security (Chief of Security)"
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_head.dm:123]:
```suggestion
	display_name = "headwear, security (chief of security)"
```

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 33]
```diff
@@ -119,19 +119,19 @@
...
+	description = "A selection of Chief of Security headwear."
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_head.dm:124]:
```suggestion
	description = "A selection of chief of security headwear."
```

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 34]
```diff
@@ -119,19 +119,19 @@
...
 
-/datum/gear/head/hos/New()
+/datum/gear/head/cos/New()
 	..()
 	var/list/hoshead = list()
-	hoshead["head of security hat, standard"] = /obj/item/clothing/head/hos
-	hoshead["head of security hat, corporate"] = /obj/item/clothing/head/hos/corp
-	hoshead["head of security beret, standard"] = /obj/item/clothing/head/beret/security/hos
-	hoshead["head of security beret, corporate"] = /obj/item/clothing/head/beret/security/hos/corp
+	hoshead["Chief of Security hat, standard"] = /obj/item/clothing/head/cos
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_head.dm:131]:
```suggestion
	hoshead["chief of security hat, standard"] = /obj/item/clothing/head/cos
```

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 35]
```diff
@@ -119,19 +119,19 @@
...
+	hoshead["Chief of Security hat, corporate"] = /obj/item/clothing/head/cos/corp
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_head.dm:132]:
```suggestion
	hoshead["chief of security hat, corporate"] = /obj/item/clothing/head/cos/corp
```

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 36]
```diff
@@ -119,19 +119,19 @@
...
+	hoshead["Chief of Security beret, standard"] = /obj/item/clothing/head/beret/security/cos
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_head.dm:133]:
```suggestion
	hoshead["chief of security beret, standard"] = /obj/item/clothing/head/beret/security/cos
```

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 37]
```diff
@@ -119,19 +119,19 @@
...
+	hoshead["Chief of Security beret, corporate"] = /obj/item/clothing/head/beret/security/cos/corp
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_head.dm:134]:
```suggestion
	hoshead["chief of security beret, corporate"] = /obj/item/clothing/head/beret/security/cos/corp
```

File: code/modules/client/preference_setup/loadout/loadout_suit.dm [hunk 38]
```diff
@@ -257,16 +257,16 @@ datum/gear/suit/colorvest
 	gear_tweaks += new /datum/gear_tweak/path(secjacket)
 
 /datum/gear/suit/secjackethos
-	display_name = "security jacket selection (Head of Security)"
-	path = /obj/item/clothing/suit/storage/security/hos
-	allowed_roles = list("Head of Security")
+	display_name = "security jacket selection (Chief of Security)"
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_suit.dm:260]:
```suggestion
	display_name = "security jacket selection (chief of security)"
```

File: code/modules/clothing/under/jobs/security.dm [hunk 39]
```diff
@@ -150,11 +150,11 @@
...
-	desc = "It's a jumpsuit worn by those few with the dedication to achieve the position of \"Head of Security\". It has additional armor to protect the wearer."
-	name = "head of security's uniform"
+	desc = "It's a jumpsuit worn by those few with the dedication to achieve the position of \"chief of security\". It has additional armor to protect the wearer."
+	name = "Chief of Security's uniform"
```
Comment (SleepyGemmy) [inline code/modules/clothing/under/jobs/security.dm:157]:
```suggestion
	name = "chief of security's uniform"
```

File: code/modules/clothing/under/miscellaneous.dm [hunk 40]
```diff
@@ -342,15 +342,15 @@
 	worn_state = "crdress"
 
 /obj/item/clothing/under/hosformalmale
-	name = "head of security's formal uniform"
-	desc = "A male head of security's formal-wear, for special occasions."
+	name = "chief of security's formal uniform"
+	desc = "A male Chief of Security's formal-wear, for special occasions."
```
Comment (SleepyGemmy) [inline code/modules/clothing/under/miscellaneous.dm:346]:
```suggestion
	desc = "A male chief of security's formal-wear, for special occasions."
```

File: code/modules/clothing/under/miscellaneous.dm [hunk 41]
```diff
@@ -342,15 +342,15 @@
...
 	icon_state = "hos_formal_male"
 	item_state = "r_suit"
 	worn_state = "hos_formal_male"
 
 /obj/item/clothing/under/hosformalfem
-	name = "head of security's formal uniform"
-	desc = "A female head of security's formal-wear, for special occasions."
+	name = "chief of security's formal uniform"
+	desc = "A female Chief of Security's formal-wear, for special occasions."
```
Comment (SleepyGemmy) [inline code/modules/clothing/under/miscellaneous.dm:353]:
```suggestion
	desc = "A female chief of security's formal-wear, for special occasions."
```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

File: code/modules/client/preference_setup/loadout/loadout_head.dm (hunk 37 above)
Comment (KingOfThePing) [reply code/modules/client/preference_setup/loadout/loadout_head.dm:134]: oh my fucking god

File: code/modules/client/preference_setup/loadout/loadout_suit.dm (hunk 38 above)
Comment (KingOfThePing) [reply code/modules/client/preference_setup/loadout/loadout_suit.dm:260]: This is not correct. This shows, e.g. in the loadout, in brackets who this stuff is allowed to spawn on. THERE it is capitalized (as seen on the removed lines)

File: code/modules/client/preference_setup/loadout/loadout_head.dm (hunk 33 above)
Comment (KingOfThePing) [reply code/modules/client/preference_setup/loadout/loadout_head.dm:124]: Loadout, it's capitalized there, so it stays that way.

File: code/modules/client/preference_setup/loadout/loadout_head.dm (hunk 34 above)
Comment (KingOfThePing) [reply code/modules/client/preference_setup/loadout/loadout_head.dm:131]: Ditto

File: code/modules/client/preference_setup/loadout/loadout_head.dm (hunk 35 above)
Comment (KingOfThePing) [reply code/modules/client/preference_setup/loadout/loadout_head.dm:132]: Ditto

File: code/modules/clothing/under/miscellaneous.dm (hunk 40 above)
Comment (KingOfThePing) [reply code/modules/clothing/under/miscellaneous.dm:346]: There is no consistency. Descriptions are capitalized, it's a job title.

**Comment by KingOfThePing** [comment]
There is hardly any consistency what is to be capitalized and what not.

This is my understanding:
Item names = all lower case
descriptions = Capitalized because it's a /job title/.
Loadout/lists = Capitalized, because see above.
Internal/backend stuff or names = all lower case.

Is this the correct way to do it, because people review something, and then the next person comes and want it changed back, since in their opinion [...]

You see the problem.

**Review by Arrow768 — CHANGES_REQUESTED** [review]
Discussed by the maintainers.

The title should be changes to `Chief Security Officer`, so its consistent with the other on station jobs. (Namely the `Chief Medical Officer` and the `Chief Engineer`)

The director level job can be dealt with by renaming the existing `Chief Security Officer` to  `Chief Security Director` (so that job is consistent with all the other director level positions)

**Comment by KingOfThePing** [comment]
I am a bit confused or I am misunderstanding you.

Head of Security = Chief Security Officer
 
???? = Chief Security Director?

I need clarification on this because I am too dumb to grasp what you wrote.

**Comment by FayeLovesU** [comment]
Chief Security Officer sounds really, really terrible. Like some kind of middle management, second in command role.

**Comment by Arrow768** [comment]
> I am a bit confused or I am misunderstanding you.
> 
> Head of Security = Chief Security Officer
> 
> ???? = Chief Security Director?
> 
> I need clarification on this because I am too dumb to grasp what you wrote.

If the Head of Security is changed, it should be changed to Chief Security Officer.
However we already have a Chief Security Officer [in our lore](https://wiki.aurorastation.org/index.php?title=Nanotrasen_Corporation) (who is the director of all security operations)
To solve that issue the existing "Chief Security Officer" is renamed to "Chief Security Director" to be in line with the other director titles.

**Comment by KingOfThePing** [comment]
Cannot finish this.

**Pull request closed without merging**